skorch/net.py

```diff
@@ -730,10 +730,12 @@ def fit_loop(self, X, y=None, epochs=None, **fit_params):
                 yi_res = yi if not y_train_is_ph else None
                 self.notify('on_batch_begin', X=Xi, y=yi_res, training=True)
                 step = self.train_step(Xi, yi, **fit_params)
+                train_batch_count += 1
+                if not step:
```

Comment:
Does `train_step` return a falsy value for this to trigger?

Reply:
What do you mean? That we should check for `if step is not None`?

Reply:
The examples in this PR that overwrite `train_step` we return something.

Reply:
Yes, you are right. This was a remnant of my earlier attempt. Fixed now.